.github/workflows/phase2-recording-integration.yml

```
@@ -0,0 +1,130 @@
+
+      - name: Apply recording integration
+        shell: bash
+        run: |
+          python - <<'PY'
+          from pathlib import Path
+
+          app = Path('src/App.tsx')
+          text = app.read_text(encoding='utf-8')
+          old = '''  const handleSaveRecordingToPlaylist = (recording: AudioRecording, targetTrackIndex: number) => {
+    const newClip: PlaylistClip = {
+      id: `rec-clip-${Date.now()}`,
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)